src/utils/codeowners.ts

```diff
@@ -44,7 +44,8 @@ const filterGeneratedContent = (content: string) => {
 export const createOwnersFile = async (
   outputFile: string,
   ownerRules: ownerRule[],
-  groupSourceComments = false
+  groupSourceComments = false,
+  customRegenerationCommand: string | undefined = undefined
```

Comment:
```suggestion
  customRegenerationCommand?: string
```
why giving it a default?

Reply:
I didn't want to change the order of the arguments, but yeah that's not a biggie. But that's why I needed an initializer (since `groupSourceComments` has one).

Reply:
so, groupSourceComments has one because is a boolean. but a string doesn't really have a state that we can default to, and undefined is already set.

Reply:
Yes, but it's also that TypeScript requires initializers on all following arguments if a previous one got one. So, in order to remove the initializer from `customRegenerationCommand`, I had to move it in the argument order to before `groupSourceComments`. But yeah, when that's solved it's as you say, `undefined` is already set. 👍